_data/banking.yml

```
@@ -961,32 +1203,42 @@ websites:
       - proprietary
       - hardware
     doc: https://www.norisbank.de/service/zwei-faktor-authentifizierung.html
+    regions:
+      - de
 
   - name: Northeast Credit Union
     url: https://necu.org
     img: necu.png
     twitter: NortheastCU
     facebook: northeastcu
+    regions:
+      - us
 
   - name: Nubank
     url: https://www.nubank.com.br
     img: nubank.png
     twitter: nubankbrasil
     facebook: nubankbrasil
     lang: pt
+    regions:
+      - br
 
   - name: Openbank
     url: https://www.openbank.es
     img: openbank.png
     tfa:
       - sms
     doc: https://www.openbank.es/en/security
+    regions:
+      - es
```

```suggestion
    regions:
      - es
      - de
      - nl
      - pt
```
Openbank also has locations in [Germany](https://www.openbank.de/), [the Netherlands](https://www.openbank.nl/), and [Portugal](https://www.openbank.pt/). While each has its own local URL, they all use SMS 2FA. We should either list all regions or change the name to show the Spain website separate from the others.